Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -90,6 +91,7 @@ def StyleError.errorMessage (err : StyleError) : String := match err with
       "In the past, importing 'Lake' in mathlib has led to dramatic slow-downs of the linter (see \
       e.g. mathlib4#13779). Please consider carefully if this import is useful and make sure to \
       benchmark it. If this is fine, feel free to allow this linter."
+  | StyleError.duplicateImports => "Duplicate imports"
```

Comment:
How about including a `name` of a duplicate import in the error message?

Reply:
On the naming side, I'd call the variant `duplicateImport` (singular).
Perhaps it's also nicer to start the error message in lower-case, but as all other variants use upper-case also, that change should be made separately for all variants.

Reply:
I think @grunweg meant the capitalisation of the String messages, not the constructor names!

Reply:
So let's just revert this change of yours and stick to 

```
| StyleError.duplicateImports => "Duplicate imports"
```

as you had it originally!